iree/turbine/kernel/kernels.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2024 The IREE Authors

```

Comment:
Maybe we can put it in `iree/turbine/kernel/wave/templates/igemm.py` or `iree/turbine/kernel/wave/templates/conv.py`. Here are some of my thoughts/ideas behind this:

1. Reasoning under `iree/turbine/kernel/wave/` because it is specifically a `wave` kernel and with the current state `iree/turbine/kernel/` is not wave exclusive. 
2. Under `iree/turbine/kernel/wave/template` because it will be somewhat uniform to the way shark-ai stack is organizing their kernels/template kernels. [(example link)](https://github.com/nod-ai/shark-ai/tree/main/sharktank/sharktank/kernels)
3. Should be in it's own file `igemm.py`/`conv.py`, because in the future, we may store different variants of igemm or conv (such as igemm, direct convolution, backward convs, etc). In a similar fashion that in the future, we may have `attention.py` that has different variants such as vanilla, paged, quantized, evoform.

Reply:
Good idea, done.